shub/config.py

```diff
@@ -182,7 +182,7 @@ def get_target(self, target, auth_required=True):
 
 MIGRATION_BANNER = """
 -------------------------------------------------------------------------------
-Welcome to shub v1.6!
+Welcome to shub 2.0!
```

Comment:
I was trying to avoid the impression that this is a welcome message to one specific version, whereas it is actually a welcome message to "shub configured through sh.yml" (e.g. v2+). But "2.0" is indeed not optimal either and could become confusing later. Maybe "v2"?